app/presenters/document_export_presenter.rb

```diff
@@ -12,60 +12,127 @@ def as_json
 
   def editions
     model.editions.map do |edition|
-      edition_associations(edition)
+      {
+        edition: edition,
+        government: edition.government,
+        whitehall_admin_links: resolve_whitehall_admin_links(edition),
+        associations: {
+          attachments: complete_attachments(edition),
+          authors: stripped_authors(edition),
+          edition_organisations: complete_organisations(edition),
+          edition_policies: stripped_edition_policies(edition),
+          images: complete_images(edition),
+          last_author: stripped_last_author(edition),
+          topical_events: strippped_topical_events(edition),
+          translations: complete_translations(edition),
+          specialist_sectors: stripped_specialist_sectors(edition),
+        },
+      }.merge(provide_doctype_information(edition))
     end
   end
 
-  def edition_associations(edition)
-    output = {
-               edition: edition,
-               associations: {},
-             }
-
-    associations = edition.class.reflect_on_all_associations.map(&:name)
-    associations.each do |association|
-      if association == :images
-        edition_images = edition.public_send(association)
-        output[:associations][association] = complete_images_hash(edition_images)
-      elsif association == :attachments
-        edition_attachments = edition.public_send(association)
-        output[:associations][association] = complete_attachments_hash(edition_attachments)
-      else
-        output[:associations][association] = edition.public_send(association)
-      end
+  def stripped_last_author(edition)
+    author = edition.try(:last_author)
+    return {} unless author
+
+    { id: author.id, uid: author.uid }
```

Comment:
Do we need both the `id` and the `uid`? That feels strange to me 🤔

Reply:
`id` is the field local to whitehall's db and uid is what's shared between applications. In signon the uid is used to identify a user however I agree that having the `id` may be overkill as probably don't need it. (however according to Kevins checklist having the id seems a required field).

Reply:
I think there are going to be some places where we're just getting user ids not uids so may need a mechanism to look them up - the versions table comes to mind. 

It could well be that some of the local Whitehall ids are superfluous but I think in general they offer quite a lot of look-up information that is not present on some of these global ids.